source/clear-linux/tutorials/tutorial-proxy.rst

```diff
@@ -0,0 +1,103 @@
+.. _tutorial-proxy:
+
+Setting up proxy
+################
+
+This tutorial shows you how to configure your system for use behind a
+corporate proxy. 
+
+|CL| :ref:`tooling` applications already benefit from the :ref:`autoproxy`
+feature. 
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Prerequisites
+*************
+
+This tutorial assumes you have installed |CL| on your host system.
+For detailed instructions on installing |CL| on a bare metal system, visit
+the :ref:`bare metal installation guide <bare-metal-install-desktop>`.
+
+Shells and programs in a desktop session
+****************************************
+
+Terminal
+========
+
+Add the following to your ~/.bashrc:
+
+.. code-block:: bash
+
+    export http_proxy=http://your.http-proxy.url:port
+    export https_proxy=http://your.https-proxy.url:port
+    export no_proxy=".your-company-domain.com,localhost"
```

Comment:
Also need to include the capitalized versions of the env variables:
export HTTP_PROXY=$http_proxy
same for HTTPS_PROXY and NO_PROXY.  You can also add variables for fttp_proxy/FTP_PROXY and socks_proxy/SOCKS_PROXY

Reply:
This cover it?

export http_proxy=http://your.http-proxy.url:port
export https_proxy=http://your.https-proxy.url:port
export ftp_proxy=http://your.ftp-proxy.url:port
export socks_proxy=http://your.socks-proxy.url:port
export no_proxy=".your-company-domain.com,localhost"
export HTTP_PROXY=$http_proxy
export HTTPS_PROXY=$https_proxy
export FTP_PROXY=$ftp_proxy
export SOCKS_PROXY=$socks_proxy
export NO_PROXY=$no_proxy

Reply:
Yep, that's how I do it